JASP-R-Interface/jaspResults/src/jaspResults.cpp

```diff
@@ -67,19 +79,24 @@ jaspResults::jaspResults(std::string title, Rcpp::RObject oldState)
 	{
 		Rcpp::Environment::global_env()["RStorageEnv"] = Rcpp::Environment::global_env().new_child(true);
 		_RStorageEnv = new Rcpp::Environment(Rcpp::Environment::global_env()["RStorageEnv"]);
+
+		if(_writeSealRoot + _writeSealRelative == "")
+			throw std::runtime_error("Write seal location not given and we are running in JASP, this should never happen!");
 	}
 	else
 		_RStorageEnv = new Rcpp::Environment(Rcpp::as<Rcpp::Environment>(Rcpp::Environment::namespace_env("jaspResults")[".plotStateStorage"]));
 
-	if(!oldState.isNULL() && Rcpp::is<Rcpp::List>(oldState))
+	bool imNotReincarnatedAfterBeingMurdered = lastWriteWorked();
+
+	if(imNotReincarnatedAfterBeingMurdered && !oldState.isNULL() && Rcpp::is<Rcpp::List>(oldState))
```

Comment:
It will happen automatically because it won't be in the "KeepList" and the tempfiles delete everything not in it every time anyway.
And because the check happens inside jaspResults and the state is still loaded through older code from `common.R` I figured just doing it like this would be fine.